policy/modules/contrib/sssd.te

```diff
@@ -284,7 +284,7 @@ optional_policy(`
 #
 
 # allow sssd_t to kill unresponsive selinux_child process
-allow sssd_t sssd_selinux_manager_t:process signal;
+allow sssd_t sssd_selinux_manager_t:process { setcap signal };
```

Comment:
Is it really 'sssd_t' that needs to get `setcap`?

I'm not fluent with selinux policies, but looks like it should be 'sssd_selinux_manager_t' (below)